evaluators/text/BleuEvaluator/__init__.py

```diff
@@ -70,14 +57,5 @@ def evaluate(self,
         # set everything to undercase and tokenize
         desired_list = desired.lower().split()
         actual_list = actual.lower().split()
+        return self.get_score([desired_list], actual_list, len([desired_list]))
```

Comment:
try this:

```
>>> desired = "hey I am here"
>>> desired_list = desired.lower().split()
>>> desired_list
['hey', 'i', 'am', 'here']
>>> len([desired_list])
1
>>> 
```
len([desired_list]) will always return 1. Why are the `n_grams` computed from the `desired_list` and not the `actual_list`?

Reply:
You're right, it was checking the list of the list. 
I changed the second test to use a n-gram less than 4, so we see the difference between resetting the weights or not.